app/views/slides/templates/_standard.html.erb

```diff
@@ -9,15 +9,21 @@
 </div>
 
 <% if slide.background_type == 'video' %>
-  <video id="js-background-video" src="<%= slide.background %>" class="ui-slide-background" preload='auto' loop></video>
+<%# `controls muted` attribute mutes videos automatically
+    Source: https://www.w3schools.com/tags/att_video_muted.asp %>
+  <!-- <video controls muted id="js-background-video" src="<%= slide.background %>" class="ui-slide-background" preload='auto' loop></video> -->
+
+  <%= video_background %>
```

Comment:
I think the helper can be ever more helpful. Imagine an interface:

    <%= slide_video_tag(slide, :background) =>